packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -134,13 +133,19 @@ class SnackBar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     assert(animation != null);
+    ThemeData theme = Theme.of(context);
+    ThemeData darkTheme = new ThemeData(
+      brightness: Brightness.dark,
+      accentColor: theme.accentColor,
+      accentColorBrightness: theme.accentColorBrightness
+    );
     List<Widget> children = <Widget>[
```

Comment:
Since you have the Theme here, should we also put the theme into the widget tree? That way the content can get to it?